examples/medical-center/persist/entity.bal

```diff
@@ -0,0 +1,18 @@
+import ballerina/time;
+import ballerina/persist as _;
+
+type MedicalItem record {|
+    readonly int itemId;
+    string name;
+    string itemType;
+    string unit;
+|};
+
+type MedicalNeed record {|
+    readonly int needId;
+    int itemId;
+    int beneficiaryId;
+    time:Civil period;
+    string urgency;
+    int quantity;
+|};
```

Comment:
```suggestion
|};

```

Reply:
updated in [29c72ce](https://github.com/ballerina-platform/persist-tools/pull/120/commits/29c72ce16d2edf4882433055270167e57360d0f7)